css/css-backgrounds/border-image-shorthand-003.htm

```diff
@@ -21,17 +27,14 @@
                 border-image-outset: 2;
                 border-image-repeat: stretch;
                 border-image-slice: 40% 15% 20% 5%;
-                border-image-source: url("../support/9grid40-30-20-10-lime.png");
+                border-image-source: url("support/9grid40-30-20-10-lime.png");
```

Comment:
This test should also use the red version.

Reply:
Yes.